src/api-requests/contentstack_utils.ts

```diff
@@ -0,0 +1,46 @@
+import { UnderlyingClient } from '../types/client-factory';
+import { DocumentNode } from 'graphql';
+
+type gqlParents =
+    | 'l1Products'
```

Comment:
This is used in `fetchAll()` and may need to expand for missing parents, which matches to the top-level key in a gql:
```
videos: all_videos(where: { slug: $slug }, skip: $skip) {
    total
    items {
        ...VideoFields
    }
}
```